HotelBooking/HotelBooking.DataAccessLayer/Entities/Room.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+public class Room
```

Comment:
Tutaj też przydałoby się int RoomId, chyba że potraktujemy numer pokoju jako unikalna wartość. Zostawiam Ani decyzję. Myślę sobie tylko czy potem patrząc na bazę będzie to intuicyjne, że Number jest kluczem. Jak mówiłem zostawiam Ani :)